import goldException
class errorHandler:
  errors=[]
  nexterror=0
  created="no"
  def _init_(self):
    created="yes"
  def 
